### ISSUE TITLE
Relations shown several times in the list of relations for an object type

### ISSUE THREAD

**Issue by martinnec** [opendata-mvcr/ontoGrapher#360]
In "OFN Časové okamžiky a intervaly" (https://slovník.gov.cz/datový/pracovní-prostor/pojem/metadatový-kontext/instance-1764959588), tab "Časový interval". When I choose "Časový okamžik" and then "Vazby", I can see a given relationship type shown several times:

![image](https://user-images.githubusercontent.com/4235208/147885272-0259b7a8-8269-4540-9871-65572152c5f3.png)

**Comment by bindeali** [comment]
I've just started #361 which hopefully addresses your previous issues. As for this, I copied this workspace to my local DB this morning and I saw a few duplicates (not as many as you have, though), however those duplicates actually exist and the quickest remedy on build #361 was to delete them and they hadn't appeared again. When #361 is merged and published I will notify you.

**Comment by martinnec** [comment]
Yes, I can also see in the workspace that there are several duplicities (e.g. in "má vztažený prvek 1/2"). Even if I remove them, they appear again after reloading the workspace.

**Comment by martinnec** [comment]
It seems OK now. I thing, that it can be closed @bindeali .
